srv/notifyToConsole.js

```diff
@@ -1,34 +1,29 @@
 const NotificationService = require('./service');
 const cds = require("@sap/cds");
 const LOG = cds.log('notifications');
-const { buildNotification, doesKeyExist } = require("./../lib/utils");
 
 module.exports = class NotifyToConsole extends NotificationService {
   async init() {
-    // call NotificationService's init
-    await super.init();
-  }
-
-  notify() {
+    await super.init()
```

Comment:
move `super.init()` behind `this.on()`?

Reply:
Yes, in general that's what I would do... just didn't know in this case whether the Outbox base class requires that before → do you know?

Reply:
no, before or after is fine. audit-logging does it last as well.